templates/terraform/examples/target_grpc_proxy_basic.tf.erb

```diff
@@ -0,0 +1,87 @@
+resource "google_compute_target_grpc_proxy" "default" {
+  name    = "tf-test-target-grpc-proxy%{random_suffix}"
```

Comment:
Can you assign the `name` value through `vars` like most of the rest of these configs do?